src/services/ui/e2e/tests/home/index.spec.ts

```diff
@@ -1,21 +1,37 @@
 import { test, expect } from "@playwright/test";
+import { chromium } from "@playwright/test";
 import * as $ from "@/selectors";
 
+const user = {
+  userName: "george@example.com",
+  password: "bigTUNA1!"
+};
+
 test("has title", async ({ page }) => {
-  await page.goto("/");
+  await page.goto("https://mako-dev.cms.gov/");
```

Comment:
we dont want to hardcode this, we should be using an env variable here, that way we can also do local host / val / ephem branches / etc

Reply:
@benjaminpaige @jdinh8124 I have made the changes just testing the changes. Thanks guys.